corehq/motech/openmrs/repeaters.py

```diff
@@ -217,6 +217,160 @@ def send_request(self, repeat_record, payload):
         return response
 
 
+class SQLOpenmrsRepeater(SQLCaseRepeater):
+    """
+    ``OpenmrsRepeater`` is responsible for updating OpenMRS patients
+    with changes made to cases in CommCare. It is also responsible for
+    creating OpenMRS "visits", "encounters" and "observations" when a
+    corresponding visit form is submitted in CommCare.
+
+    The ``OpenmrsRepeater`` class is different from most repeater
+    classes in three details:
+
+    1. It has a case type and it updates the OpenMRS equivalent of cases
+       like the ``CaseRepeater`` class, but it reads forms like the
+       ``FormRepeater`` class. So it subclasses ``CaseRepeater`` but its
+       payload format is ``form_json``.
+
+    2. It makes many API calls for each payload.
+
+    3. It can have a location.
+
+    """
+    class Meta(object):
+        proxy = True
+        app_label = 'repeaters'
+
+    friendly_name = _("Forward to OpenMRS")
+    payload_generator_classes = (FormRepeaterJsonPayloadGenerator,)
+
+    include_app_id_param = OptionValue(default=False)
+    location_id = OptionValue(default='')
+
+    _has_config = True
+
+    # self.white_listed_case_types must have exactly one case type set
+    # for Atom feed integration to add cases for OpenMRS patients.
+    # self.location_id must be set to determine their case owner. The
+    # owner is set to the first CommCareUser instance found at that
+    # location.
+    atom_feed_enabled = OptionValue(default=False)
+    atom_feed_status = OptionValue(obj_schema=AtomFeedStatus)
+    openmrs_config = OptionValue(obj_schema=OpenmrsConfig)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
This can be removed.
```suggestion
```

Reply:
https://github.com/dimagi/commcare-hq/pull/30348/commits/0c3d199773b4f80641470faec7f3a1f5a8a9dae1